command/ca/ca.go

```diff
@@ -192,3 +194,18 @@ func completeURL(rawurl string) (string, error) {
 	// rawurl looks like ca.smallstep.com:443 or ca.smallstep.com:443/1.0/sign
 	return completeURL("https://" + rawurl)
 }
+
+// BetaCommand enables access to beta APIs.
+func BetaCommand() cli.Command {
+	return cli.Command{
+		Name:      "ca",
+		Usage:     "commands that are made available for testing new features and APIs",
+		UsageText: "step beta ca <subcommand> [arguments] [global-flags] [subcommand-flags]",
```

Comment:
`** step beta ca**` I've noticed another error like this before, there might be other commands with this problem.